wireless/pktradio/pktradio_metadata.c

```diff
@@ -49,7 +49,7 @@ static FAR struct pktradio_metadata_s *g_free_metadata;
 
 /* Supports mutually exclusive access to the free list */
 
-static sem_t g_metadata_sem;
+static mutex_t g_metadata_lock;
```

Comment:
```suggestion
static mutex_t g_metadata_lock = NXMUTEX_INITIALIZER;
```

Reply:
next patch

Reply:
Sorry. I missed the call of `nxmutex_init(&g_metadata_lock);` and posted this comment. Now reviewed again and all is in place